src/Resources/config/menu.xml

```diff
@@ -18,7 +18,8 @@
             <tag name="knp_menu.provider"/>
         </service>
         <service id="sonata.admin.menu.matcher.voter.admin" class="Sonata\AdminBundle\Menu\Matcher\Voter\AdminVoter" public="true">
-            <tag name="knp_menu.voter" request="true"/>
+            <argument type="service" id="request_stack" />
```

Comment:
There is no constructor argument for this: https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Menu/Matcher/Voter/AdminVoter.php